webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/index.js

```diff
@@ -0,0 +1,139 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Card,
+  CardHeader,
+  CardTitle,
+  CardBody,
+  ClipboardCopy,
+  Divider,
+} from '@patternfly/react-core';
+import { UserIcon } from '@patternfly/react-icons';
+import { foremanUrl } from '../../../common/helpers';
+import { translate as __ } from '../../../common/I18n';
+import SkeletonLoader from '../../common/SkeletonLoader';
+import { STATUS } from '../../../constants';
+
+import './styles.css';
+
+const DetailsCard = ({
+  ip,
+  ip6,
+  mac,
+  comment,
+  owner_id: ownerID,
+  owner_name: ownerName,
+  hostgroup_name: hostgroupName,
+  hostgroup_id: hostgroupID,
+  status,
+}) => (
+  <Card isHoverable>
+    <CardHeader>
+      <CardTitle>{__('Details')}</CardTitle>
+    </CardHeader>
+    <CardBody>
+      <DescriptionList
+        isAutoColumnWidths
+        columnModifier={{
+          default: '2Col',
+        }}
+      >
+        <DescriptionListGroup>
+          <DescriptionListTerm>{__('IPv6 Address')}</DescriptionListTerm>
+          <DescriptionListDescription>
+            <SkeletonLoader status={status}>
+              {ip6 && (
+                <ClipboardCopy variant="inline-compact">{ip6}</ClipboardCopy>
+              )}
+            </SkeletonLoader>
+          </DescriptionListDescription>
+        </DescriptionListGroup>
+        <DescriptionListGroup>
+          <DescriptionListTerm>{__('IPv4 Address')}</DescriptionListTerm>
+          <DescriptionListDescription>
+            <SkeletonLoader status={status}>
+              {ip && (
+                <ClipboardCopy variant="inline-compact">{ip}</ClipboardCopy>
+              )}
+            </SkeletonLoader>
+          </DescriptionListDescription>
+        </DescriptionListGroup>
+        <DescriptionListGroup>
+          <DescriptionListTerm>{__('Mac Address')}</DescriptionListTerm>
+          <DescriptionListDescription>
+            <SkeletonLoader status={status}>
+              {mac && (
+                <ClipboardCopy variant="inline-compact">{mac}</ClipboardCopy>
+              )}
+            </SkeletonLoader>
+          </DescriptionListDescription>
+        </DescriptionListGroup>
+        <DescriptionListGroup>
+          <DescriptionListTerm>{__('Host Group')}</DescriptionListTerm>
+          <DescriptionListDescription>
+            <SkeletonLoader status={status}>
+              {hostgroupID && (
+                <a href={foremanUrl(`/hostgroups/${hostgroupID}/edit`)}>
```

Comment:
I wonder if we should have links here and on the user to their edit pages? users might not have the needed permissions, and since these items don't have a show page than clicking will lead to an unexpected edit form.